### PR TITLE
WIP:2021 04 05 adrian01c daily branch

### PR DESCRIPTION
**Pull request by Adrian01C**

### CODE REVIEW

**Review by QuintyHH — APPROVED** [review]
The commit messages have nothing to do with the actual code changes :) Please try and be more descriptive of what your commit is about.

Approved this time around

**Review by simonamarcu2 — APPROVED** [review]
Nice work! 💯

**Review by biggicode — APPROVED** [review]

File: web/adrian01c/grid/plic.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  .gridul-meu {
+    grid-template-columns: repeat(8, 1fr);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .gridul-meu {
+    grid-template-columns: repeat(12, 1fr);
+  }
+}
```
Comment (vasilemaftita) [inline web/adrian01c/grid/plic.css:40]: I understand that space is missing, I see that space 41 is missing!

**Review by vasilemaftita — COMMENTED** [review]
Nice: Work!

**Pull request merged**